development/input-app/components/input/Forwardrefinput/Forwardrefinput.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from "react";
```

Comment:
Rename these files to `ForwardRefInput` Follow the same pattern used in other examples.